applications/new_schema.py

```diff
@@ -37,9 +35,7 @@ class Meta:
         exclude = ("berthapplication_set",)
 
     def resolve_harbor(self, info, **kwargs):
-        if self.harbor:
-            return self.harbor.servicemap_id
-        return None
+        return self.harbor.servicemap_id
```

Comment:
update also the field on this Type to be `required=True` 🙂

Reply:
Uhh, good catch!